homeassistant/components/vicare/const.py

```diff
@@ -21,8 +21,6 @@
     "Heatbox2_SRC",
     "E3_TCU10_x07",
     "E3_TCU41_x04",
-    "E3_FloorHeatingCircuitChannel",
```

Comment:
why are they supported now?

Reply:
That PR is still open. Why are these removed from UNSUPPORTED_DEVICES in this PR and not on that one?